### ISSUE TITLE
0.9.0 is incompatible with official rabbitmq Docker image

### ISSUE THREAD

**Issue by cap10morgan** [rabbitmq/rabbitmq-autocluster#54]
* RabbitMQ version: 3.6.12
* Erlang version: 19.2.1

When running version 0.9.0 in the official `rabbitmq:3.6.12-management` Docker image, it fails to start with this error:

```
Failed to enable plugin "autocluster": it may have been built with an incompatible (more recent?) version of Erlang
```

Would it be possible to release a version of 0.9.0 that is compatible with this Docker image?

**Comment by michaelklishin** [comment]
@cap10morgan mind posting server logs so that we are not basing our thinking on a single line that has "may" in it?

**Comment by michaelklishin** [comment]
We are actually releasing a 0.10.0 today. I can build it on Erlang 18 but it should be possible to use any other version with the Docker image.

**Comment by michaelklishin** [comment]
Here's a 0.10.0 (tip of stable) built on OTP 18.3:

[rabbitmq-autocluster-0.10.0.zip](https://github.com/rabbitmq/rabbitmq-autocluster/files/1399681/rabbitmq-autocluster-0.10.0.zip)

**Comment by michaelklishin** [comment]
I released [0.10.0](https://github.com/rabbitmq/rabbitmq-autocluster/releases/tag/0.10.0) built on Erlang/OTP 18.3.

**Comment by cap10morgan** [comment]
Thanks, 0.10.0 works great!
